molecule/default/playbook.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: Converge
+  hosts: all
+
+  pre_tasks:
+    - name: Ensure nginx Directories Exist
```

Comment:
Perhaps we should include the Sansible nginx role as a dependency? Might be better than munging a directory. You could include the users and groups role as well so we don't have to use the php user for fpm.